sos/plugins/yum.py

```diff
@@ -44,6 +44,13 @@ def setup(self):
         # Get a list of channels the machine is subscribed to.
         self.add_cmd_output("yum -C repolist")
 
+        # Get a list of available plugins
+        cmd = """bash -c '
+            for i in /usr/lib/yum-plugins/*.py; do
+                echo "$i ($(rpm -qf $i))";
+            done'"""
```

Comment:
Please don't put shell code in `sos`: there's absolutely no reason to not do this in python. It may also be better to simply collect the whole of `/etc/yum/pluginconf.d` since each plugin is required to have a configuration file there which will also reveal the enable states.